pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[tool.poetry]
+name = "ixctl"
+repository = "https://github.com/fullctl/ixctl"
+version = "0.1.0"
+description = "Full control of your environment"
+authors = ["20C <code@20c.com>"]
+license = "Apache-2.0"
+
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Build Tools",
+    "Topic :: Software Development :: Documentation",
```

Comment:
Not "build tools" or "documentation" -- can delete both lines

Reply:
Fixed in https://github.com/fullctl/ixctl/pull/38/commits/659f65d46701275c329e6588c6f4351224771d77